src/components/sbb-link/sbb-link.tsx

```diff
@@ -153,19 +160,12 @@ export class SbbLink implements LinkButtonProperties, ComponentInterface {
     }
   }
 
-  /**
-   * Generate the class attribute based on component's parameters.
-   */
-  private _getClassString(): string {
-    const textSizeClass = this.variant === 'inline' ? '' : ` sbb-link--text-${this.textSize}`;
-    const iconPositionClass =
-      this.iconPlacement === 'start'
-        ? ' sbb-link--icon-placement-start'
-        : ' sbb-link--icon-placement-end';
-    const inlineClass = this.variant === 'inline' ? ' sbb-link--inline' : '';
-    const negativeClass = this.negative ? ' sbb-link--negative' : '';
-
-    return `sbb-link${textSizeClass}${iconPositionClass}${inlineClass}${negativeClass}`;
+  /** State of listed named slots, by indicating whether any element for a named slot is defined. */
+  @State() private _namedSlots = createNamedSlotState('icon');
```

Comment:
Please move this property to the other properties

Reply:
done